Sources/StreamChatUI/ChatMessageList/MessageComposer/ChatMessageComposerSuggestionsViewController.swift

```diff
@@ -273,9 +278,9 @@ open class _ChatMessageComposerSuggestionsMentionDataSource<ExtraData: ExtraData
         ) as! _ChatMessageComposerMentionCollectionViewCell<ExtraData>
 
         let user = searchController.users[indexPath.row]
-        // We need to make sure we set the uiConfig before accessing the mentionView,
-        // so the mentionView is created with the most up-to-dated uiConfig.
-        cell.uiConfig = uiConfig
+        // We need to make sure we set the components before accessing the mentionView,
+        // so the mentionView is created with the most up-to-dated components.
+        cell.components = components
```

Comment:
So... There is no way of avoiding this passing of `components` to the cell? :(

Reply:
I think we may do some extra magic like if the view is a subclass of TableViewCell or CollectionViewCell, ask for a super view of TableView or CollectionView class, I didn't do any investigation around this and if we want, we should threaten it as a separated task :)